ui/src/app/app-routing.module.ts

```diff
@@ -102,7 +102,7 @@ const routes: Routes = [
                 component: WorkspaceEditComponent,
                 data: {
                   title: 'Clone Workspace',
```

Comment:
Update title

Reply:
Good catch

Reply:
Q: should we also update the `clone` path?